buildSrc/src/main/kotlin/publish.gradle.kts

```diff
@@ -49,4 +57,24 @@ afterEvaluate {
             }
         }
     }
+
+    signing {
+        val signingKey = System.getenv("ORG_GPG_PASSPHRASE")
+        val signingPassword = System.getenv("ORG_GPG_PRIVATE_KEY")
+        useInMemoryPgpKeys(signingKey, signingPassword)
+
+        useGpgCmd()
+        sign(publishing.publications)
+    }
+}
+
+gradle.taskGraph.whenReady {
```

Comment:
Can we rewrite this more like this so it does not loop through all the tasks even if it already found the correct task?
Maybe also add some println for information that the signing is disabled here :)

```
gradle.taskGraph.whenReady {
    tasks.withType(Sign::class) {
        
        val publishToMavenLocalTask = allTasks.find { "publishToMavenLocal" in it.name }
        var isPublishingToMavenLocal = publishToMavenLocalTask != null

        onlyIf { !isPublishingToMavenLocal }
    }
}
```